scripts/test-seedbox-maps.sh

```
@@ -0,0 +1,78 @@
+    preset.yaml: |
+      layers: {}
+EOF
+
+helm template test "$chart" -f "$tmp/inline.yaml" > "$tmp/inline-rendered.yaml"
+grep -Fq "mountPath: \"/custom-maps/$uuid\"" "$tmp/inline-rendered.yaml"
+grep -Fq "checksum/maps:" "$tmp/inline-rendered.yaml"
+test "$(grep -Fc "checksum/maps:" "$tmp/inline-rendered.yaml")" -eq 1
+
+helm template test "$chart" -f "$tmp/inline.yaml" \
+  --show-only templates/configmap.yaml > "$tmp/app-config.yaml"
+! grep -Fq "customerUUID:" "$tmp/app-config.yaml"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)